index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+   <nav class="top-header">
+     <a class="welcome identical-to" href="#">Welcome</a>
+      <span class="identical-to">&#8801;</span>
+  </nav>
+  </header> 
+  <main>
+    <section>
+    <div class="intro">
+    <h1 class="first-name">Hey there. I'm Stephen</h1>
+    <h1 class="first-name software-dev">I'm a software developer</h1>
+  </div>
+    <p class="paragraph">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hestiate to contact me.</p>    
+    <ul class="social">
+      <li><img class="social-img" src="images/twitter.png" alt="twitter"></li>
+      <li><img class="social-img" src="images/Linkedin.png" alt="linkedin"></li>
+      <li><img class="social-img" src="images/medium.png" alt="gmail"></li>
+      <li><img class="social-img" src="images/github.png" alt="github"></li>
+      <li><img class="social-img" src="images/angellist.png" alt="angellist"></li>        
+  </ul>
```

Comment:
- [ ] Great that you have used `<li>` tags as per the suggestion from previous reviewer. However, you should remove the little dots in front of the icons by adding `list-style: none` to your `.social` css class (the `ul` tag
![social-media-icons](https://user-images.githubusercontent.com/44057863/164884376-77c4ad8d-48cf-44fd-a4ab-bc7b11b432ed.png)
)

- [ ] You should also wrap the icons with `<a>` tag to make them clickable and take the user to visit your corresponding social media profiles. e.g 
```
<ul>
  <li>
   <a href="#">
      <img src ="">
    </a>
   </li>
</ul>
```

- [ Optional ] You may also add your social medial links the `href` property of the `<a>` tags.

Reply:
Thanks but you did'nt refer to my latest commit, there is not dot there. Here is my code. You agree with me that  list-style-type: none; removed the dot. Check the lates file please. Thanks.
.social {
  display: flex;
  justify-content: space-between;
  margin-bottom: 32px;
  list-style-type: none;
}

Nevethless, I have made the social icons clickable by wrapping them in a tag. Thanks!